packages/react-charting/src/components/DonutChart/Arc/Arc.tsx

```diff
@@ -61,7 +71,34 @@ export class Arc extends React.Component<IArcProps, IArcState> {
           aria-label={this._getAriaLabel()}
           role="img"
         />
+
         {this._renderArcLabel(classNames.arcLabel)}
+
```

Comment:
enable this section only if gradients are available

Reply:
Resolved here and in GuageChart